qiskit/transpiler/passes/routing/stochastic_swap.py

```diff
@@ -291,7 +291,7 @@ def _mapper(self, circuit_graph, coupling_graph, trials=20):
         for i, v in enumerate(layerlist):
             logger.debug("    %d: %s", i, v["partition"])
 
-        qubit_subset = self.trivial_layout.get_virtual_bits().keys()
+        qubit_subset = self.trivial_layout._v2p.keys()
```

Comment:
This I believe is hit only once per call to `StochasticSwap.run()`. Is the speed up here worth the private access?

Reply:
Fixed in: https://github.com/Qiskit/qiskit-terra/pull/7036/commits/009b005c51541207781bc3f0828cdca202486f1d